components/alibi-explain-server/Dockerfile

```diff
@@ -15,19 +15,17 @@ ARG PYTHON_VERSION
 ARG CONDA_VERSION
 
 # Install Python / Conda
-RUN conda install --yes python=${PYTHON_VERSION} conda=${CONDA_VERSION}
-RUN pip install pip==21.2.4 setuptools==58.1.0
-RUN dnf install -y make automake gcc gcc-c++
+RUN conda install --yes python=${PYTHON_VERSION} conda=${CONDA_VERSION} && \
```

Comment:
if we opted for `-c conda-forge` in alibi detect image why not here?

Reply:
Desperation after trying several things out.

I'll add it there though for consistency :+1: